src/components/Dashboard/widgets/PieChart.tsx

```diff
@@ -1,19 +1,54 @@
 import React, { ReactElement } from 'react';
+import { PieChart } from 'react-minimal-pie-chart';
+import { useSelector } from 'react-redux';
 
 import styled from '@emotion/styled';
 
-import { WidgetContainer, WidgetTitle } from './common';
+import { selectAssets } from 'store/selectors';
+import { randomColor } from 'util/helpers';
+import { Config } from 'vars/defines';
+
+import { WidgetContainer } from './common';
 
 const PieChartRoot = styled(WidgetContainer)`
   grid-column: span 2;
+  padding: 12px;
 `;
 
-const PieChart = (): ReactElement => {
+const CustomPieChart = styled(PieChart)`
+  position: relative;
+  z-index: 1;
+  .desc {
+    font-size: var(--font-size-tiny-p);
+    fill: var(--color-white);
+  }
+  .small {
+    fill: var(--color-gray);
+    font-size: var(--font-size-tinytiny-p);
+  }
+`;
+const PieChartWdiget = (): ReactElement => {
+  const assets = useSelector(selectAssets);
+  const totalValue = assets.reduce((total, { amount, price }) => total + amount * price, 0);
```

Comment:
This gotta be centralized later as it is calculated in two places already.

Reply:
Also Widget is misspelled.